tests/java/test-app/src/main/java/com/godaddy/asherah/testapp/regression/DynamoDbGlobalTableTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+Currently disabled because DynamoDbLocal isn't playing nice with maven-shade-plugin.
```

Comment:
Could extend this comment to include a `TODO` regarding the "placeholder" nature of the tests and the plans to relocate the tests in general